coverage.json

```diff
@@ -0,0 +1 @@
sing_lines": 0, "excluded_lines": 0}, "missing_lines": [], "excluded_lines": []}, "keras_nlp/src/utils/transformers/convert_gemma.py": {"executed_lines": [14, 16, 17, 18, 19, 20, 23, 24, 25, 27, 36, 38, 61, 64, 66, 72, 73, 75, 80, 81, 86, 87, 91, 92, 97, 98, 104, 115, 126, 137, 150, 155, 160, 167, 172, 175, 176, 177, 178, 179, 180, 181, 182, 183, 186, 187], "summary": {"covered_lines": 46, "num_statements": 46, "percent_covered": 100.0, "percent_covered_display": "100", "missing_lines": 0, "excluded_lines": 0}, "missing_lines": [], "excluded_lines": []}, "keras_nlp/src/utils/transformers/convert_gpt2.py": {"executed_lines": [14, 16, 17, 18, 19, 20, 23, 24, 35, 37, 41, 47, 48, 51, 55, 59, 63, 69, 72, 79, 88, 95, 104, 111, 120, 127, 133, 140, 144, 151, 157, 161, 166, 169, 170, 171, 172, 173, 174, 175, 176, 177, 180, 181, 182, 183], "summary": {"covered_lines": 46, "num_statements": 46, "percent_covered": 100.0, "percent_covered_display": "100", "missing_lines": 0, "excluded_lines": 0}, "missing_lines": [], "excluded_lines": []}, "keras_nlp/src/utils/transformers/convert_llama3.py": {"executed_lines": [14, 16, 17, 18, 19, 22, 23, 33, 34, 38, 45, 46, 49, 50, 52, 56, 62, 67, 72, 77, 86, 92, 98, 106, 111, 114, 115, 116, 117, 118, 119, 120, 121, 122, 125, 126, 127, 128, 130, 131, 133, 134, 136], "summary": {"covered_lines": 43, "num_statements": 43, "percent_covered": 100.0, "percent_covered_display": "100", "missing_lines": 0, "excluded_lines": 0}, "missing_lines": [], "excluded_lines": []}, "keras_nlp/src/utils/transformers/convert_pali_gemma.py": {"executed_lines": [14, 16, 17, 18, 19, 20, 23, 24, 25, 26, 47, 51, 54, 59, 66, 72, 77, 83, 84, 86, 91, 96, 101, 106, 112, 117, 123, 128, 133, 139, 144, 150, 155, 161, 166, 173, 181, 189, 190, 193, 197, 203, 214, 225, 236, 249, 254, 259, 266, 272, 278, 281, 282, 283, 284, 285, 286, 287, 288, 289, 292, 303], "summary": {"covered_lines": 62, "num_statements": 62, "percent_covered": 100.0, "percent_covered_display": "100", "missing_lines": 0, "excluded_lines": 0}, "missing_lines": [], "excluded_lines": []}, "keras_nlp/src/utils/transformers/safetensor_utils.py": {"executed_lines": [14, 16, 17, 18, 19, 20, 22, 23, 28, 29, 30, 32, 39, 40, 41, 43, 44, 45, 47, 61, 62, 64, 65, 69, 70, 72, 73, 74, 76, 79, 81, 82, 84, 85, 88, 90, 91, 93, 94, 95, 96, 97], "summary": {"covered_lines": 42, "num_statements": 47, "percent_covered": 89.36170212765957, "percent_covered_display": "89", "missing_lines": 5, "excluded_lines": 0}, "missing_lines": [24, 25, 33, 66, 67], "excluded_lines": []}, "keras_nlp/src/version_utils.py": {"executed_lines": [15, 18, 21, 22], "summary": {"covered_lines": 4, "num_statements": 5, "percent_covered": 80.0, "percent_covered_display": "80", "missing_lines": 1, "excluded_lines": 0}, "missing_lines": [23], "excluded_lines": []}}, "totals": {"covered_lines": 10066, "num_statements": 11377, "percent_covered": 88.47675134042366, "percent_covered_display": "88", "missing_lines": 1311, "excluded_lines": 9}}
```

Comment:
This looks autogenerated? And it I don't see any workflows to continue to generate it.

If we want to check-in coverage, we should do the same thing Keras does.

See https://github.com/keras-team/keras/blob/master/codecov.yml and https://app.codecov.io/gh/keras-team/keras

I don't think we should commit a json file like this.